src/formio/components/FileField.js

```diff
@@ -189,6 +189,18 @@ class FileField extends Formio.Components.components.file {
     super.upload(files);
   }
 
+  validatePattern(file, val) {
+    // file type not recognized; get file extension and specify mime type manually
+    if (file.type === '') {
+      let ext = file.name.split('.').pop();
+      if (ext === 'msg') {
+        return val.includes('application/vnd.ms-outlook');
+      }
```

Comment:
I wouldn't shortcut validation here, since wildcards can be supported in formio. Instead, I'd fixup the actual file information:

```
file.type = 'application/vnd.ms-outlook';
```